daml-foundations/daml-ghc/ide/test/Development/IDE/State/API/Testing.hs

```diff
@@ -231,7 +231,7 @@ parseShakeProfileJSON testDir json =
     , Aeson.Array entry <- V.toList entries
      -- Number == 0, built in the last run
     , Aeson.String name : _ : Aeson.Number 0 : _ <- [V.toList entry]
-    , Just res <- [stripInfix "; " $ replace (testDir ++ "/") "" $ T.unpack name]
+    , Just res <- [stripInfix "; " $ replace (testDir ++ [FilePath.pathSeparator]) "" $ T.unpack name]
```

Comment:
```suggestion
    , Just res <- [stripInfix "; " $ replace (testDir </>) "" $ T.unpack name]
```

Reply:
`</> ""` doesn't seem to add a trailing slash, so I left the `++ [FilePath.pathSeparator]`

Reply:
You may wish to use `addTrailingPathSeparator`. Or you might not, really up to you - whatever suits.

Reply:
Thanks, I missed this one! This is nice as it will not duplicate if the one is already in `testDir`